File: 2265-PartitionArrayAccordingToGivenPivot/2265-PartitionArrayAccordingToGivenPivot.cpp
// Last updated: 6/18/2026, 4:20:46 PM
class Solution {
public:
    vector<int> pivotArray(vector<int>& nums, int pivot) {
        vector<int>ans;
        for(int i : nums) if(i < pivot) ans.push_back(i);
        for(int i : nums) if(i == pivot) ans.push_back(i);
        for(int i : nums) if(i > pivot) ans.push_back(i);
        return ans;
    }
};

File: 2265-PartitionArrayAccordingToGivenPivot/2265-PartitionArrayAccordingToGivenPivot.java
// Last updated: 6/18/2026, 4:20:43 PM
class Solution {
    public int[] pivotArray(int[] nums, int pivot) {
        int []ans = new int[nums.length];
        int k = 0;
        for(int i : nums) if(pivot > i) ans[k++] = i;
        for(int i : nums) if(pivot == i) ans[k++] = i;
        for(int i : nums) if(pivot < i) ans[k++] = i;
        return ans;
    }
}